### PR TITLE
Start setting up tui tests

### PR DESCRIPTION
**Pull request by matthewmturner**

### CODE REVIEW

File: tests/util.rs
Comment (matthewmturner) [inline tests/util.rs:1]: I guess if these are specific to cli this actually isnt needed. So i may just remove this file

**Comment by matthewmturner** [comment]
@alamb FYI this is to start setting up the structure to test the tui.

File: src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,4 @@
+pub mod app;
```
Comment (alamb) [inline src/lib.rs:1]: 👍

File: tests/util.rs
Comment (alamb) [reply tests/util.rs:1]: We can move them in a follow on PR

**Review by alamb — APPROVED** [review]
I ran all the tests locally and it looks good to me!

**Comment by alamb** [comment]
Since we are early I think we should just do as much dev on `main` as possible to keep up velocity. I'll try and find time soon to do the follow ons from https://github.com/datafusion-contrib/datafusion-tui/pull/120

**Pull request merged**